packages/tocco-util/src/notifier/components/TitleMessage.js

```diff
@@ -1,32 +1,41 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import {FormattedMessage} from 'react-intl'
-import {FormattedValue} from 'tocco-ui'
+import {FormattedValue, Typography} from 'tocco-ui'
 import _isString from 'lodash/isString'
 
+const containsHtml = s => s && /<\/?[a-zA-Z0-9]*\/?>/.test(s)
 const isKey = s => s && s.startsWith('client.')
 
-const Content = ({content}) =>
-  _isString(content)
-    ? (isKey(content) ? <FormattedMessage id={content}/> : <FormattedValue type="html" value={content}/>)
-    : content
+/**
+ * Format content correctly
+ * @param  {string|object} content can be a React component, a translation key or string may containing html
+ * @param  {boolean} isTitle wraps non html strings into H4 or P
+ * @return {object} React component
+ */
+const Content = ({content, isTitle}) => {
+  if (_isString(content)) {
+    if (containsHtml(content)) {
+      content = <FormattedValue type="html" value={content}/>
+    } else {
+      content = isKey(content) ? <FormattedMessage id={content}/> : content
+      content = isTitle ? <Typography.H4>{content}</Typography.H4> : <Typography.P>{content}</Typography.P>
+    }
+  }
+  return content
+}
 
 Content.propTypes = {
   content: PropTypes.node
 }
 
-const TitleMessage = ({title, message, className, children}) => {
+const TitleMessage = ({title, message, children}) => {
   return (
-    <div className={className}>
-      <header className="tocco-notifier__title">
-        <Content content={title}/>
-      </header>
-      {message
-      && <div className="tocco-notifier__message">
-        <Content content={message}/>
-      </div>}
+    <React.Fragment>
+      {title && <Content content={title} isTitle={true} />}
```

Comment:
Content "is Title"? Rather confusing. IMO it would be better to introduce e Title component than creating a component that is kind of both.

I think is considered bad practice to reasigne a property (in this case content)

Reply:
I have replaced `props.isTitle` by `props.tag` to make component `Content` more versatile. Having two components which shares complete codebase next to a single tag (e.g. <h4> or <p>) seems awkward to me.

I have introduced variable output.